config.json

```diff
@@ -1296,13 +1296,13 @@
         "player_warp_energy_multiplier": 0.12,
         "use_warp_energy_for_cloak": true,
         "warp_energy_multiplier": 0.12,
-        "warp_memory_effect": 0.9,
+        "warp_memory_effect": 0.5,
```

Comment:
This setting controls how much of your ship’s speed is saved and restored when dropping out of SPEC and then reentering SPEC mode. This setting was much too high; 0.5 seems to be a better value.